runtime/oti/JITInterface.hpp

```diff
@@ -235,6 +280,8 @@ class VM_JITInterface
 		return (void*)((J9JITRegisters*)vmThread->entryLocalStorage->jitGlobalStorageBase)->gpr.numbered[30]; // LR
 #elif defined(J9VM_ARCH_S390)
 		return (void*)((J9JITRegisters*)vmThread->entryLocalStorage->jitGlobalStorageBase)->gpr.numbered[14];
+#elif defined(J9VM_ARCH_RISCV)
+		return (void*)((J9JITRegisters*)vmThread->entryLocalStorage->jitGlobalStorageBase)->gpr.named.RA; // RA
```

Comment:
Please remove the (now) redundant comments (e.g. `// RA`).

Reply:
Removed.